alignak/objects/service.py

```diff
@@ -301,6 +301,14 @@ def get_name(self):
             return self.name
         return 'SERVICE-DESCRIPTION-MISSING'
 
+    def get_servicegroups(self):
```

Comment:
Yes, why not ... but there is so much functions like this one. We should organize another code cleaning session to fix all this kind of functions later.